include/tss2/tss2_tcti.h

```diff
@@ -24,6 +49,9 @@ typedef struct pollfd TSS2_TCTI_POLL_HANDLE;
 #elif defined(_WIN32)
 #include <windows.h>
 typedef HANDLE TSS2_TCTI_POLL_HANDLE;
+#elif defined (__VXWORKS__)
+#include <sys/poll.h>
+typedef struct pollfd TSS2_TCTI_POLL_HANDLE;
```

Comment:
Since this is the same as for linux, unix, apple and qnx, I'd go ahead and extend that ifdef instead of defining a new one.
Or at least move this elif up there to directly follow the similar ones.